### ISSUE TITLE
JSON export, missingHeadings and WCAG numbers

### ISSUE THREAD

**Issue by etoscano** [microsoft/accessibility-insights-web#5746]
I am looking at the JSON export of Accessibility Insights For Web and I see that "requirementKey": "missingHeadings" is connected to        
"wcagNumber": "1.3.1" "wcagName": "Info and Relationships"   
and to 
"wcagNumber": "2.4.1" "wcagName": "Bypass Blocks"

However, when I read the information regarding test 5.2 No missing headings, this test is connected to 1.3.1: Info and Relationships and 2.4.6: Headings and Labels, and not to 2.4.1 Bypass Blocks.

**Comment by JGibson2019** [comment]
Hi @etoscano can you provide the link to the website you're testing with so we can investigate/attempt to repro on our end?

**Comment by ferBonnin** [comment]
> Hi @etoscano can you provide the link to the website you're testing with so we can investigate/attempt to repro on our end?

@JGibson2019 I don't think we need a link to a website as this repos with any site when exporting as JSON.

**Comment by lisli1** [comment]
@etoscano, thanks for using Accessibility Insights for Web and catching this issue! The "missingHeadings" test should point to WCAG 1.3.1 and 2.4.6. We will fix the discrepancy in the JSON export data.

**Comment by lisli1** [comment]
@etoscano, the fix is now live in Canary Version 2022.7.27.1622. At your earliest convenience, please verify that the issue has been resolved.
